testflight/suite_test.go

```diff
@@ -133,6 +135,35 @@ var _ = AfterEach(func() {
 	fly("destroy-pipeline", "-n", "-p", pipelineName)
 })
 
+func downloadFly(atcUrl string) (string, error) {
+	var flyPath string
+	client := concourse.NewClient(atcUrl, http.DefaultClient, false)
+	readCloser, _, err := client.GetCLIReader("amd64", runtime.GOOS)
+	if err != nil {
+		return flyPath, err
+	}
+	cwd, err := os.Getwd()
```

Comment:
Might it make more sense to create a temporary file rather than putting fly in the working directory? This could prevent unexpectedly removing a file named `fly` (not sure how likely that scenario is, but still - I would be surprised if it happened to me).

You can use `ioutil.TempFile` for this